pkgs/development/python-modules/nbqa/default.nix

```
@@ -0,0 +1,64 @@
+{ autopep8
+, black
+, buildPythonPackage
+, fetchPypi
+, flake8
+, ipython
+, isort
+, jupytext
+, lib
+, mypy
+, pylint
+, pythonAtLeast
+, pythonOlder
+, pyupgrade
+, ruff
+, tokenize-rt
+, tomli
+}:
+buildPythonPackage rec {
+  pname = "nbqa";
+  version = "1.7.0";
```

```suggestion
  version = "1.7.0";
  format = "setuptools";
```
Please add a format explicitly.

---

`format ? "setuptools"`, so this isn't necessary.

---

There are over 2300 packages that define `format = "setuptools"`.
e.g. `rg 'format = "setuptools"' pkgs/development/python-modules/ -l | wc -l`
It is set by default, but it is preferable to set it explicitly.

---

There's even more packages which use the default:

```console
$ rg --files-without-match 'format = ' pkgs/development/python-modules/*/default.nix | wc -l
2759
```